app/src/main/res/values-zh-rCN/mozonline_strings.xml

```diff
@@ -18,7 +18,7 @@
     <!-- Privacy Notice positive button-->
     <string name="privacy_notice_positive_button">同意并继续</string>
     <!-- Privacy Notice neutral button-->
-    <string name="privacy_notice_neutral_button">退出应用</string>
+    <string name="privacy_notice_neutral_button">不同意并退出</string>
```

Comment:
For translation reasons, we don't update strings, we just create a new string with a sequential number for example `privacy_notice_neutral_button_1` and mark it for future [removal](https://github.com/mozilla-mobile/fenix/wiki/Removing-strings), you could find an example [here](https://github.com/mozilla-mobile/fenix/commit/888bbc770eb1ecd781262fa7c3a5a8b632aeb6fe#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103)

Reply:
Thanks! I think the strings in mozonline_strings.xml will not be translated into more languages, but I'll update to keep consistency.